### PR TITLE
Feature 3.4/swagger 3.25.1

### PR DESCRIPTION
**Pull request by Simran-B**
Backport of #11530

### CODE REVIEW

**Comment by Simran-B** [comment]
The magic of cherry-picking... wasn't really my plan to backport the doc expansion change, but I think it's for the better?

File: js/server/assets/swagger/index.html [hunk 1]
```diff
@@ -1,132 +1,111 @@
...
+          SwaggerUIBundle.presets.apis,
+          SwaggerUIStandalonePreset
+        ],
+        plugins: [
+          SwaggerUIBundle.plugins.DownloadUrl
+        ],
+        layout: "StandaloneLayout",
+        //#region ArangoDB-specific changes
+        validatorUrl: null,
+        docExpansion: "none",
```
Comment (pluma) [inline js/server/assets/swagger/index.html:90]: I'd consider this one a regression. Can you please change this to `"list"` to match the existing 3.4 behavior?

Comment (Simran-B) [reply js/server/assets/swagger/index.html]: Restored to "list"

**Comment by KVS85** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/10018/

http://172.16.10.101:8080/view/Documentation/job/arangodb-ANY-examples/626/

**Comment by KVS85** [comment]
Tests blue except unrelated failure.

**Pull request merged**